extensions/shared/upgrade-nudge/style.scss

```diff
@@ -0,0 +1,85 @@
+@media ( max-width: 600px ) {
+	// stylelint-disable-next-line selector-max-id
+	.is-iframed #wpbody {
+		padding-top: 0;
+	}
+}
+
+@media ( min-width: 600px ) {
+	body.is-iframed.is-fullscreen-mode .edit-post-header {
+		top: 0;
+	}
+}
+
+@media ( min-width: 782px ) {
+	.edit-post-layout__content {
+		position: fixed;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		top: 88px;
+		min-height: calc( 100% - 88px );
+		height: auto;
+		margin-left: 160px;
+	}
+
+	body.is-fullscreen-mode .edit-post-layout__content {
+		top: 56px;
+		position: fixed;
+	}
+}
+
+@media ( min-width: 782px ) and ( max-width: 1079px ) {
+	.is-fullscreen-mode .edit-post-layout.has-fixed-toolbar .components-notice {
+		top: 37px;
+	}
+}
+
+.components-notice-list {
```

Comment:
This seems scary :)

Reply:
Thanks, good spot. Accidentally copied over from Automattic/wp-calypso#33723, removed per ae89afc.